scripts/clean-cluster-kind.sh

```diff
@@ -9,9 +9,11 @@ set -x
 
 source scripts/common.sh
 
-IMAGE=$1
-if ${RUNTIME} exec -ti kind-control-plane crictl inspecti ${IMAGE} &> /dev/null ; then
-	${RUNTIME} exec -ti kind-control-plane crictl rmi ${IMAGE}
-fi
-
+for image in "$@"; do
+	if ${RUNTIME} exec -ti kind-control-plane crictl inspecti ${image} &> /dev/null ; then
+		echo "Delete image ${image}"
+		${RUNTIME} exec -ti kind-control-plane crictl rmi ${image}
+	fi
+done
 kubectl delete deploy cocl-operator -n confidential-clusters || true
+kubectl delete deploy register-server -n confidential-clusters || true
```

Comment:
This should be deleted by the operator. We haven't really define how the uninstallation works yet. But you should bind the registration deployment to the confidential cluster crd, when this is deleted then you should also delete the registration deployment by the operator. Only when the child objects are deleted we can remove the operator